### ISSUE TITLE
Can't build image

### ISSUE THREAD

**Issue by espoal** [meteorhacks/meteord#95]
Hi guys,

i'm using ubuntu 15.10 for my host, and I'm trying to build your image, anyhow it fails.

Could it be due to a locale issue? I'm trying to find a workaround for [this](https://github.com/meteor/meteor/issues/4019).

```
root@lalande:/ozone/dockerfiles/meteord# docker build -t ozonetmt .
Sending build context to Docker daemon 2.048 kB
Step 1 : FROM meteorhacks/meteord:onbuild
# Executing 2 build triggers...
Step 1 : COPY ./ /app
 ---> Using cache
Step 1 : RUN bash $METEORD_DIR/on_build.sh
 ---> Running in 99f7b45992bd
Downloading Meteor distribution

Meteor 1.3.2.2 has been installed in your home directory (~/.meteor).
Writing a launcher script to /usr/local/bin/meteor for your convenience.

To get started fast:

  $ meteor create ~/my_cool_app
  $ cd ~/my_cool_app
  $ meteor

Or see the docs at:

  docs.meteor.com

build: You're not in a Meteor project directory.

To create a new Meteor project:
  meteor create <project name>
For example:
  meteor create myapp

For more help, see 'meteor --help'.
The command '/bin/sh -c bash $METEORD_DIR/on_build.sh' returned a non-zero code: 1
```

**Comment by jshimko** [comment]
Did you definitely run this from a Meteor app directory?

**Comment by espoal** [comment]
no, the folder is called meteord to highlight that it contains the dockerfile of meteord. Meteor or nodejs are non installed on the host.

**Comment by jshimko** [comment]
I think you might be misunderstanding how this image is intended to be used.  It's a base image for building _your_ Meteor app in a Docker container.  You create a Dockerfile in the root of a Meteor app with `FROM meteorhacks/meteord:onbuild` in it, then run the build command.  This project doesn't create an app for you - it copies yours into the container.

**Comment by espoal** [comment]
No I think I got it right. I was trying simply to build the container, but I got the error. Should I add a meteor app to make it work?  I can't generate the container without an app?

**Comment by morfarpeter** [comment]
I get the same problem on my staging server. Not on my live server. If i try building image on the small staging I get the error, but runs fine on the other. Does that make any sense?

**Comment by Flavien-Pensato** [comment]
Your Meteor version is above the 1.3 ?

Try this image: abernix/meteord:onbuild
